.taskcluster.yml

```
@@ -167,41 +168,62 @@ tasks:
                   command:
-                      - run-task
-                      - '--taskcluster-checkout=/builds/worker/checkouts/src'
-                      - '--task-cwd=/builds/worker/checkouts/src'
-                      - '--'
-                      - bash
-                      - -cx
-                      - >
-                        ln -s /builds/worker/artifacts artifacts &&
-                        ~/.local/bin/taskgraph decision
-                        --pushlog-id='0'
-                        --pushdate='0'
-                        --project='${project}'
-                        --message=""
-                        --owner='${ownerEmail}'
-                        --level='${level}'
-                        --repository-type=git
-                        --target-tasks-method=taskcluster-branches
-                        --tasks-for='${tasks_for}'
-                        --base-repository='${baseRepoUrl}'
-                        --base-ref='${base_ref}'
-                        --head-repository='${repoUrl}'
-                        --head-ref='${head_ref}'
-                        --head-rev='${head_sha}'
-                        --head-tag='${head_tag}' &&
-                        rm artifacts/actions.json # taskcluster is not using actions.json
+                      - - bash
+                        - "-cx"
+                        - |-
+                          podman run --name taskcontainer --add-host=taskcluster:127.0.0.1 --net=host \
+                              -e "REPOSITORIES=$${REPOSITORIES}" \
+                              -e "RUN_ID=$${RUN_ID}" \
+                              -e "TASKCLUSTER_BASE_REF=$${TASKCLUSTER_BASE_REF}" \
+                              -e "TASKCLUSTER_BASE_REPOSITORY=$${TASKCLUSTER_BASE_REPOSITORY}" \
+                              -e "TASKCLUSTER_HEAD_REF=$${TASKCLUSTER_HEAD_REF}" \
+                              -e "TASKCLUSTER_HEAD_REPOSITORY=$${TASKCLUSTER_HEAD_REPOSITORY}" \
+                              -e "TASKCLUSTER_HEAD_REV=$${TASKCLUSTER_HEAD_REV}" \
+                              -e "TASKCLUSTER_HEAD_TAG=$${TASKCLUSTER_HEAD_TAG}" \
+                              -e "TASKCLUSTER_PIP_REQUIREMENTS=$${TASKCLUSTER_PIP_REQUIREMENTS}" \
+                              -e "TASKCLUSTER_PROXY_URL=$${TASKCLUSTER_PROXY_URL}" \
+                              -e "TASKCLUSTER_PULL_REQUEST_NUMBER=$${TASKCLUSTER_PULL_REQUEST_NUMBER}" \
+                              -e "TASKCLUSTER_REPOSITORY_TYPE=$${TASKCLUSTER_REPOSITORY_TYPE}" \
+                              -e "TASKCLUSTER_ROOT_URL=$${TASKCLUSTER_ROOT_URL}" \
+                              -e "TASKCLUSTER_WORKER_LOCATION=$${TASKCLUSTER_WORKER_LOCATION}" \
+                              -e "TASK_ID=$${TASK_ID}" \
+                              "mozillareleases/taskgraph:decision-d1ddb5593679cec9d7d2cdd8e9b8e2228b14ebe9b844411276cc3cfe37d189aa@sha256:b236321d98367f9cdbbd28c47bad7c0df34884897b75d266185d2e9e569ac0fa" \
```

Now's probably a good time to bump to [latest decision image](https://hub.docker.com/layers/mozillareleases/taskgraph/decision-8af16934fd68303cebf9735705515cb48df542d251c04e921c7ac50564faac32/images/sha256-2965f42c2e91fc66ae7bd27d5dff88297225a90d04ebc6de23c538e3933d4049?context=explore)

```suggestion
                              "mozillareleases/taskgraph:decision-8af16934fd68303cebf9735705515cb48df542d251c04e921c7ac50564faac32@sha256:2965f42c2e91fc66ae7bd27d5dff88297225a90d04ebc6de23c538e3933d4049" \
```

---

Created https://github.com/taskcluster/taskcluster/pull/6068